### PR TITLE
Tower Of Hanoi using Python

### PR DESCRIPTION
**Pull request by aishwaryachand**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
YES
### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
issue #2723

### CODE REVIEW

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few changes: 
- Name the file as `Tower_of_Hanoi.py`
- Add a new line at the EOF
- Use `if __name__ == '__main__'`  to run the driver code

**Review by ankitaggarwal23 — COMMENTED** [review]
Why the code looks exactly similar from [here](https://www.geeksforgeeks.org/c-program-for-tower-of-hanoi/).

**Comment by aishwaryachand** [comment]
> Why the code looks exactly similar from [here](https://www.geeksforgeeks.org/c-program-for-tower-of-hanoi/).

I did write the code by myself, however, when I first tried to understand it, I did look at several sites to understand the rules.  Am I supposed to make any other change ?

File: Python/README.md [hunk 1]
```diff
@@ -279,5 +279,6 @@ _add list here_
 - [Dutch National Flag Algorithm Implementation in Python](other/SortColors-DutchNationalFlag.py)
 - [Ploting Diffrent Graphs in Python](other/Plot_Graph.py)
 - [Remove Minimum Number of Parenthesis for Balanced String](other/check_valid_parenthesis_string.py)
+- [Tower of Hanoi](other/Tower Of Hanoi.py)
```
Comment (ankitaggarwal23) [inline Python/README.md]: Fix the typo error `(other/Tower Of Hanoi.py)` to `(other/Tower_of_Hanoi.py)`

File: Python/other/Tower_of_Hanoi.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Input:
+n_disks : number of disks
+Output:
+result: stack of disks moved from one rod to
+        another based on the rules
+"""
+#function for tower of hanoi
+#we have three rods source , auxilary and target
+#putting n_disk from source rod  to the target rod using auxiallary rod
+def tower_of_hanoi(n_disks, source, auxiliary, target):
```
Comment (ankitaggarwal23) [inline Python/other/Tower_of_Hanoi.py:23]: Add a new line above function

File: Python/other/Tower_of_Hanoi.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+#Move disk 1 from rod X to rod Z.
+#Move disk 2 from rod X to rod Y.
+#Move disk 1 from rod Z to rod Y.
+#Move disk 3 from rod X to rod Z.
+#Move disk 1 from rod Y to rod X.
+#Move disk 2 from rod Y to rod Z.
+#Move disk 1 from rod X to rod Z.
+
+# number of moves  2n - 1.
+# Time Complexity  O(2^n)
```
Comment (ankitaggarwal23) [inline Python/other/Tower_of_Hanoi.py]: Use multiline comments

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Rest looks good to merge.

Do the mentioned changes

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**